src/Core/tests/DeviceTests/Handlers/Label/LabelHandlerTests.iOS.cs

```diff
@@ -34,6 +35,24 @@ public async Task FontFamilyInitializesCorrectly(string family)
 				Assert.NotEqual(fontManager.DefaultFont.FamilyName, nativeFont.FamilyName);
 		}
 
+		[Fact]
+		public async Task PaddingInitializesCorrectly()
+		{
+			var label = new LabelStub()
+			{
+				Text = "Test",
+				Padding = new Thickness(5, 10, 15, 20)
+			};
+
+			var handler = await CreateHandlerAsync(label);
+			var insets = ((MauiLabel)handler.NativeView).TextInsets;
+
+			Assert.Equal(5, insets.Left);
+			Assert.Equal(10, insets.Top);
+			Assert.Equal(15, insets.Right);
+			Assert.Equal(20, insets.Bottom);
+		}
+
```

Comment:
Adds an on-device test to verify that the property is being set on the native iOS control. This test will be run on an iOS device as part of the CI process.